### DISCUSSION TITLE
Caricaturando Hollywood

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#526] [Curiosidades]
Post de referência: https://www.vish.blog.br/artigos/caricaturando-hollywood/

**Comment by one-eyed-willy** [comment]
O Nicolas Cage e o Sean Penn nem precisam de uma ilusão de ótica pra se parecerem com caricaturas

**Comment by brunoaugusto** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [sábado, 30 de abril de 2022](<https://www.vish.blog.br/artigos/caricaturando-hollywood/#comment_1114000817>):

> O Nicolas Cage e o Sean Penn nem precisam de uma ilusão de ótica pra se parecerem com caricaturas

![null](<https://i.postimg.cc/2y9xhv8j/nicolas-cage-holding-laugh-then-ROTL.gif>)

**Comment by angellisy** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [sábado, 30 de abril de 2022](<https://www.vish.blog.br/artigos/caricaturando-hollywood/#comment_1114000817>):

> O Nicolas Cage e o Sean Penn nem precisam de uma ilusão de ótica pra se parecerem com caricaturas

Achei ofensivo, posta mais
